assets/data/characters/jean/en.json

```diff
@@ -180,6 +180,13 @@
       "rarity": 4,
       "price": 1680,
       "image": "outfit-sea-breeze-dandelion"
+    },
+    {
+      "type": "Alternate",
+      "name": "Gunnhildr's Legacy",
+      "description": "Jean's outfit. This classic knight's ceremonial outfit is suited to all manner of occasions.",
+      "rarity": 5,
+      "image": "outfit-gunnhildr-s-legacy"
```

Comment:
[rarity should be 4](https://genshin-impact.fandom.com/wiki/Gunnhildr%27s_Legacy)